src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -734,8 +750,10 @@ public String getBootstrapServers() {
         }
         
         if (proxyContainer != null) {
-            // returning the proxy host and port for indirect connection
-            return String.format("PLAINTEXT://%s", getProxy().getListen());
+            this.getProxy();
```

Comment:
it looks weird to me you are calling a `getXXX` method but actually not using at all what is returned. I can see it's mocking the proxy but why it's not used them? Is `getProxy` the right name for the method?

Reply:
Yes, agree... such a name is a bit misleading. The better way would be to divide the method into 2 separate ones: 
 1. `ensureProxyInitialized()`, which would be a whole mocking proxy stuff...
 2. `getProxy()`, which would just get the proxy as expected

 WDYT?